src/Sylius/Component/Order/Model/Order.php

```diff
@@ -435,7 +435,9 @@ public function getAdjustmentsTotal($type = null)
 
         $total = 0;
         foreach ($this->getAdjustments($type) as $adjustment) {
-            $total += $adjustment->getAmount();
+            if (!$adjustment->isNeutral()) {
+                $total += $adjustment->getAmount();
+            }
```

Comment:
What's the rationale behind this change? 

The only use-case I can think of for neutrality is taxes, maybe there could be other adjustments such as handling fees which could be inclusive, or commission fees?

In any case, I think the totalling individual adjustment types should not exclude neutral adjustments. If you want to find out how much of something there is on an order, you should just be able to call this method for that type.

While I appreciate that this is different from the overall total which _does_ consider neutrality, that's because it's trying to sum up different sources of amount to give an overall balance. I think it's different from focussing on a particular adjustment type.

It feels wrong that every time you have a possibly neutral adjustment type you're going to have to add your own method for `getTaxTotal()` to do it explicitly with a different implementation.

We have added quite a few helpful wrapper methods such as `getTaxTotal()` which are useful for templates, but this simply wraps a fixed `return getAdjustmentsTotalRecursively(Adjusment::TAX_ADJUSTMENT);`, it doesn't require a separate implementation.